addons/event/models/event_ticket.py

```diff
@@ -148,3 +148,10 @@ def is_launched(self):
             return ticket.start_sale_date <= current_date
         else:
             return True
+
+    @api.ondelete(at_uninstall=False)
+    def check_registrations_when_delete(self):
```

Comment:
Seems like naming convention is like "_unlink_except ..." . Could be named like _unlink_except_if_registrations or something similar.

Reply:
Done, changed to _unlink_except_if_registrations